### ISSUE TITLE
Sayit and Sonos: Web instance and device cannot be selected in config dialog

### ISSUE THREAD

**Issue by t4qjXH8N** [ioBroker/ioBroker.sayit#49]
Problem: After selecting "Sonos" no devices or web instance can be selected, i.e. no devices or instances are shown. 

Additional informations on installed adapters:
```
Web server 2.4.0
Sonos 1.7.4
SayIt 1.7.0 or 1.6.8 (both do not work)
Admin3
Js-controller 1.4.2
iobroker running on Windows 10
```

Installing SayIt 1.6.7 solves the problem, devices and instances can be selected. This version seems to use the old Admin configuration dialog.

**Comment by GermanBluefox** [comment]
Can you please provide screenshot?

**Comment by t4qjXH8N** [comment]
Should be fixed in #50.
